whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
-render the distribution completely uniform would require moving all
-observing angles an average of $\simeq 30$ degrees, or 300 complete
-rotations per night.  The timing of this has not been considered.
-Whether or not this uniformity could be achieved with less slew time
-if implemented in scheduling remains to be demonstrated.
-
-A similar metric for RotTelPos should be developed.
+The RotSkyPos metrics show that the majority of fields have good randomization
+of detector angles projected on the sky.  The randomization of parallactic
+angles is less successful, though this is to be expected due to fewer knobs
+available to adjust the parallactic angle of observations of a given field.  In
+both cases, however, a significant fraction of fields show metric values lower
+than expected for a uniform distribution.  Regardless of the \emph{per field}
+criterion adopted, it is desirable to avoid the incidence of individual
+discrepant fields.  The recommended criterion for randomization of RotSkyPos and
+parallactic angle is not the behavior of the majority of the fields, but of the
+minority with the least random behavior -- the number of non-random fields
+should be minimized.
+
+It is certain that actively controlling the statistics of RotSkyPos will require
+additional slewing of the camera rotator.  At present, the operations plan is to
+only slew (beyond that required to track the sky during exposures) when
+necessary to prepare for a filter change - that could be estimated at the
+equivalent of $\simeq 3$ complete rotations per night.  To engage the rotator by
+up to $\simeq 30$ degrees per visit would require $\simeq 300$ complete
+rotations per night, though it may be possible to increase isotropy with far
+less additional slewing.  The impact on survey efficiency and hardware wear and
+tear has not been considered.  Whether or not this uniformity could be achieved
+with less slew time if implemented in scheduling remains to be demonstrated.
+
+Increasing the isotropy of the parallactic angle is trickier, since the
+parallactic angle is only affected by the hour angle of observations (for a
+given field).  It may be possible, however, to adjust the scheduler cost
+function to better favor parallactic angle isotropy.
+
+In summary, image quality weighted randomization of RotSkyPos is required at
+10-20\% scatter in image quality over the sample of visits (2015 simulation
+mentioned above).  Randomization of parallactic angle is also desired, but a
+requirement is yet to be determined.  Interestingly, there is an excellent
+opportunity to combine these two randomizations in an efficient observing
+strategy which actually minimizes the number of camera rotations.   Re-visits to
+a field over a range of hour angles and with camera rotations assures full 180
+degree range coverage for CCD coordinates relative to sky.  A metric for this
+can be written and run with OpSim to explore optimization, including minimizing
+camera rotations.
```

Comment:
Big picture question:

These metrics focus on "using survey strategy to mitigate additive shear systematics".

Is there a reason there are no metrics developed to mitigate multiplicative shear systematics?  For example, if the depth or average seeing are very heterogeneous in their variation across the sky, this means that any multiplicative biases that have to be corrected will have severe spatial dependence.  Maybe there needs to be a metric for this?  Or is this already encapsulated in metrics for other science cases, given that survey homogeneity is generally valued?